Fix Version Notation and Add Dynamic Support for Version Categorization (#19327)
cockroachdb/docs: src/current/releases/index.md

</section>

<section class="filter-content" markdown="1" data-scope="windows">

>     Windows 8 or higher is required. Windows downloads are **experimental**. Experimental downloads are not yet qualified for production use and not eligible for support or uptime SLA commitments, whether they are for testing releases or production releases

Reviewer: Double-checking: this period was removed because it was generating double periods, right? In preview, it still shows one period at the end (which I think is correct).

To review the CCL, refer to the [CockroachDB Community License](https://www.cockroachlabs.com/cockroachdb-community-license) page. You can find the applicable Business Source License or third party licenses by reviewing these in the `licenses` folder for the applicable version of CockroachDB in the GitHub repository [cockroachdb/cockroach](https://github.com/cockroachdb/cockroach). See individual files for details.

> ## Unsupported Versions 

Reviewer: ```suggestion
## Unsupported versions 
```

See my earlier comment.

> [Here]({% link releases/unsupported-versions.md %}) are the versions of CockroachDB that are no longer supported

Reviewer: Note the extra line break to separate from the heading text.

```suggestion

Release notes for unsupported CockroachDB versions are found on [Unsupported Versions]({% link releases/unsupported-versions.md %}).
```
